numba/tests/npyufunc/test_gufunc.py

```diff
@@ -140,6 +163,25 @@ def sum_row(inp, out):
         with self.assertRaisesRegex(TypeError, msg):
             sum_row(inp)
 
+    def test_axis(self):
+        # issue https://github.com/numba/numba/issues/6773
+        @guvectorize("(n)->(n)")
```

Comment:
This isn't the exact reproducer from the ticket as there's no type signature provided?

Reply:
This is the dynamic gufunc version of it. The exact one is at https://github.com/numba/numba/pull/6786/files#diff-53626a767921908cedfa774172dffeebb669f76bb7ee26af3bc51c37987d5bfdR67, but both version are doing through the same `GUFunc.__call__`. So this is the variant to make sure that the dynamic gufunc is handling the kwargs correctly as well.

Reply:
ah thanks, missed that.